### DISCUSSION TITLE
Phlebodium x hemipinnatum

### DISCUSSION THREAD

**Discussion by choess** [pteridogroup/ppg#20] [Q&A] [answered]
Mostly leaving this here in the hopes of further information. This hybrid from Chiapas was diagnosed on morphological grounds as _Phlebodium pseudoaureum_ × _Polypodium pleurosorum_ . If confirmed, this would necessitate a new nothogenus. I'm not sure it's wise to proceed on it without some molecular evidence to back it up as well, though.

**Comment by SchneiderHarald** [comment, marked as answer]
Given the challenges of the classification among the Central American diversity of Polyplodium and relatives, it may be better to confirm the status of the hybrid using molecular data before proceeding.
